configs/kgs-bot/adversary_bot.cfg

```diff
@@ -0,0 +1,207 @@
+# rootSymmetryPruning = true
+
+# Uncomment and set to true to make KataGo avoid a particular joseki that some KataGo nets misevaluate,
+# and also to improve opening diversity versus some particular other bots that like to play it all the time.
+# avoidMYTDaggerHack = false
+
+# Have KataGo mildly prefer to avoid playing the same joseki in every corner of the board.
+# Uncomment to set to a specific value. Otherwise, defaults to 0 in even games, and to 0.005 in handicap games.
+# See also the Avoid SGF mechanism at the bottom of this config.
+# avoidRepeatedPatternUtility = 0.0
+
+# Experimental logic to make KataGo fight a bit against mirror Go even with unfavorable komi.
+# Disabled by default, uncomment and set to true to enable it.
+# antiMirror = false
+
+# Search limits-----------------------------------------------------------------------------------
+
+# For all of "maxVisits", "maxPlayouts", "maxTime", search will still try to follow GTP time controls and may make a move
+# faster than the specified max if GTP tells it that it is playing under a clock as well in the current game.
+
+# If provided, limit maximum number of root visits per search to this much. (With tree reuse, visits do count earlier search)
+
+numBots = 2
+maxVisits0 = 600
+searchAlgorithm0 = EMCTS1
+maxVisits1 = 1
+searchAlgorithm1 = MCTS
+# If provided, limit maximum number of new playouts per search to this much. (With tree reuse, playouts do not count earlier search)
+# maxPlayouts = 300
+# If provided, cap search time at this many seconds.
+# maxTime = 10
+
+# Ponder on the opponent's turn?
+ponderingEnabled = false
+# maxTimePondering = 60
+# Note: you can also set "maxVisitsPondering" or "maxPlayoutsPondering" too.
+
+# Approx number of seconds to buffer for lag for GTP time controls - will move a bit faster assuming there is this much lag per move.
+lagBuffer = 1.0
+
+# Number of threads to use in search
+numSearchThreads = 1
+
+# Play a little faster if the opponent is passing, for friendliness
+searchFactorAfterOnePass = 0.50
+searchFactorAfterTwoPass = 0.25
+# Play a little faster if super-winning, for friendliness
+searchFactorWhenWinning = 0.40
+searchFactorWhenWinningThreshold = 0.95
+
+# GPU Settings-------------------------------------------------------------------------------
+
+# Maximum number of positions to send to a single GPU at once.
+# The default value here is roughly equal to numSearchThreads, but you can specify it manually
+# if you are running out of memory, or if you are using multiple GPUs that expect to split
+# up the work.
+# nnMaxBatchSize = <integer>
+
+# Cache up to (2 ** this) many neural net evaluations in case of transpositions in the tree.
+# Uncomment and edit to change if you want to adjust a major component of KataGo's RAM usage.
+nnCacheSizePowerOfTwo = 20
+
+# Size of mutex pool for nnCache is (2 ** this).
+nnMutexPoolSizePowerOfTwo = 16
+
+numNNServerThreadsPerModel = 1
+openclDeviceToUseThread0 = 2
```

Comment:
might be able to remove this? I'm guessing you got this from Tony's Sabaki PR but he's removed it in that PR too

Reply:
Yes, you're right. I commented them out now (like other options that aren't changed from defaults).